src/components/atoms/review/ReportUserRole.vue

```diff
@@ -1,4 +1,6 @@
 <template>
+  <div>
+    <h3>This is the debug version of ReportUserRole.vue</h3>
```

Comment:
Was a debug work in progress, I re-implemented this class in `ReportUserRoleExplicit` and forgot to remove these old debug tags since this component isn't being used anymore.